client-src/overlay.js

```diff
@@ -23,10 +23,25 @@ let iframeContainerElement;
 let containerElement;
 /** @type {Array<(element: HTMLDivElement) => void>} */
 let onLoadQueue = [];
+/** @type {TrustedTypePolicy | undefined} */
+let overlayTrustedTypesPolicy;
 
 ansiHTML.setColors(colors);
 
-function createContainer() {
+/**
+ * @param {string | null} trustedTypesPolicyName
+ */
+function createContainer(trustedTypesPolicyName) {
+  // Enable Trusted Types if they are available in the current browser.
+  if (window.trustedTypes) {
+    overlayTrustedTypesPolicy = window.trustedTypes.createPolicy(
```

Comment:
In case there is an error creating a Trusted Types policy the overlay is not shown at all. 

A different approach would be to capture the exception outside this function (same place we define `overlayTrustedTypesPolicy`) and show an overlay without using `innerHTML` showing that the policy creation failed. 

However I think having an error in the console is sufficient, since this is a development only feature so we can assume the console is opened anyways.